mesh/pncmesh.cpp

```diff
@@ -1516,11 +2017,11 @@ void ParNCMesh::Refine(const Array<Refinement> &refinements)
    for (int i = 0; i < refinements.Size(); i++)
```

Comment:
```suggestion
   for (int i = 0; i < refinements.Size() && Iso; i++)
```
same as suggestion for `ParNCMesh::AnisotropicConflict`